CMakeLists.txt

```
@@ -58,26 +59,40 @@ include_directories(SYSTEM "lib/easyloggingpp/src" "lib/better-enums/")
 if (COMPILE_TESTS)
     add_subdirectory("lib/googletest")
 endif()
-if (COMPILE_PYBIND)
-    add_subdirectory("lib/pybind11")
-endif()
 
 set( CMAKE_BUILD_TYPE_COPY "${CMAKE_BUILD_TYPE}" )
 set( CMAKE_BUILD_TYPE "Release" )
 option(build_static_lib "Build easyloggingpp as a static library" ON)
-add_subdirectory("lib/easyloggingpp")
+if (BUILD_PIP_PACKAGE)
+    # EXCLUDE_FROM_ALL is undocumented and used to solve issues with easyloggingpp access rights
```

```suggestion
    # Relies on undocumented behaviour. EXCLUDE_FROM_ALL is used to prevent install commands 
    # inside the easyloggingpp CMakeLists from executing and subsequently failing with a permission error,
    # making it impossible to install the Python package as a normal user.
```
Or something like that.